devices.js

```diff
@@ -2647,6 +2647,30 @@ const devices = [
             await configureReporting.batteryPercentageRemaining(endpoint);
         },
     },
+    {
+        zigbeeModel: ['TRADFRI SHORTCUT Button'],
+        model: 'E1812',
+        vendor: 'IKEA',
+        description: 'TRADFRI Shortcut Button',
+        supports: 'on, brightness up/stop',
+        fromZigbee: [
+            fz.command_on, fz.legacy_genOnOff_cmdOn, fz.command_move,
+            fz.legacy_E1743_brightness_up, fz.command_stop, fz.legacy_E1743_brightness_stop, fz.battery,
```

Comment:
Please use the non-legacy versions, so for e.g. `legacy_E1743_brightness_up` -> `command_move`

Reply:
I have removed the "legacy" entries. Is that what you meant?